resources/src/androidMain/kotlin/dev/icerock/moko/resources/FileResourceContainer.kt

```diff
@@ -0,0 +1,7 @@
+/*
+ * Copyright 2020 IceRock MAG Inc. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package dev.icerock.moko.resources
+
+actual interface FileResourceContainer<T>
```

Comment:
why explicit `File`? i suggest to add it at all resource containers: MR.strings, MR.plurals, MR.files and others...it can be useful in later

Reply:
Renamed to `ResourceContainer`.